data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -885,11 +893,10 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
 
     <section title="Flow Aggregation Control">
       <t>
-        Flow aggregation includes aggregation accomplished through the use of
-        hierarchical LSPs in MPLS and tunnels, in the case of IP, MPLS and TSN,
-        all of which aggregate multiple DetNet flows into a single new DetNet
-        flow.  Aggregation can also be grouping of IP flows that share
-        6-tuple attributes or flow identifiers at the DetNet sub-layer.
+        Flow aggregation means that multiple App-flows are served by a single new DetNet
+        flow.  There are many techniques to achieve aggregation, for example in case of IP, 
+		it can be grouping of IP flows that share 6-tuple attributes or flow 
+		identifiers at the DetNet sub-layer.

```

Comment:
Why drop the MPLS example that are very familiar to some. Suggest adding back:

Another example includes aggregation accomplished through the use of
hierarchical LSPs in MPLS and tunnels.

Reply:
OK.